### DISCUSSION TITLE
How to config path for output dir?

### DISCUSSION THREAD

**Discussion by Centurion-Rome** [comfyanonymous/ComfyUI#917] [Q&A] [answered]
Where can I define save directory for generated images (node save image)

**Comment by ltdrdata** [comment]
https://github.com/comfyanonymous/ComfyUI/issues/875#issuecomment-1631218090

This will help you.

**Reply by Centurion-Rome** [reply, marked as answer]
thank you.Found it:
use command line param:
```
--output-directory
```

We should have somewhere documented list of possible command line arguments?
